src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const total = 40 * days;
+
+  if (days >= 7) {
+    return total - 50;
```

Comment:
You should create variables for discount

Reply:
agree